### DISCUSSION TITLE
onMouseEnter, onMouseLeave in RowProps

### DISCUSSION THREAD

**Discussion by dandelionn** [table-library/react-table-library#126] [Q&A]
Hello! I have noticed that we can only set onClick and onDoubleClick events in RowProps. Are there any plans to add onMouseEnter, onMouseLeave events? I need to be able to retrieve the id of the row when this events happen.

`export declare type RowProps<T extends TableNode> = {
    item: T;
    className?: string;
    disabled?: boolean;
    onClick?: OnClick<T>;
    onDoubleClick?: OnClick<T>;
    children: React.ReactNode;
} & RestProps;`

**Comment by murilloFlixSign** [comment]
How about this? I need an oncontextmenu event with item props.
